packages/apps/wallet/src/pages/Wallet/index.tsx

```diff
@@ -25,60 +35,54 @@ import {
   TabContent,
   TextInput,
   Container,
-} from '@origyn-sa/origyn-art-ui';
-import styled from 'styled-components';
-import { Link } from 'react-router-dom';
+} from '@origyn-sa/origyn-art-ui'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import TransferTokensModal from '@dapp/features-sales-escrows/modals/TransferTokens'
 
 const GuestContainer = () => {
-  const { logIn } = useContext(AuthContext);
+  const { logIn } = useContext(AuthContext)
 
   return (
     <Box
-      component="main"
+      component='main'
```

Comment:
Undo all of these changes here in Wallet index.tsx so only needed update is here, this has conflicts and hard to review properly...